demos/launch/office_crowd.launch.xml

```diff
@@ -0,0 +1,51 @@
+<?xml version='1.0' ?>
```

Comment:
Can we avoid creating a whole new file and just having the crowd sim be a launch parameter? Similar to what is done with ignition through the `use_ignition` launch parameter.

Reply:
I am not sure about creating a whole new file, as there could be some legacy old files left over due to merging my old version with current rmf_demos. It is possible use `use_crowdsim` launch parameter.

Reply:
That sounds strange, we use the trick of "having a plugin in the world file but not in the path" to allow models to work with both ignition and gazebo so not finding a plugin should print an error but not crash the simulation, for example we do it for differential drive robots like the [Caddy](https://github.com/osrf/rmf_demos/blob/master/rmf_demo_assets/models/Caddy/model.sdf#L31-L95).
I could reproduce the crash when the menge environment variable is not present but it seems to be related to the plugin's code rather than gazebo / ignition itself.